### DISCUSSION TITLE
ocp4 egress ips

### DISCUSSION THREAD

**Discussion by slibonati** [fabric8io/kubernetes-client#4274] [Q&A]
What is the equivalent of `oc get egressip`in OCP4  using openshift-client? The API group is k8s.ovn.org. 

client.hasAPiGroup("k8s.ovn.org", true) returns true.

I think I may have to use the CRD API but not sure how to proceed.

Can anyone point me to example code?  

Thanks!

**Comment by rohanKanojia** [comment]
I think you should be able to use it using Fabric8 Dynamic Client API ( GenericKubernetesResource): https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md#resource-typeless-api

**Comment by slibonati** [comment]
Thank you. That worked perfectly.
